packages/embark-reset/index.js

```diff
@@ -18,11 +18,15 @@ exports.paths = new Set([
 ]);
 
 exports.reset = async ({
-  doneMessage = green('reset done!'),
+  doneMessage = green('Done!'),
```

Comment:
I guess we could get rid of the `completeMessage` variable. Especially since it seems to be declared as a global(?)

Reply:
It's a function parameter with default value, not a global.